linera-chain/src/unit_tests/data_types_tests.rs

```diff
@@ -84,10 +84,10 @@ fn test_certificates() {
     let key1 = ValidatorSecretKey::generate();
     let key2 = ValidatorSecretKey::generate();
     let key3 = ValidatorSecretKey::generate();
-    let name1 = ValidatorName(key1.public());
-    let name2 = ValidatorName(key2.public());
+    let validator1_pk = key1.public();
+    let validator2_pk = key2.public();
 
-    let committee = Committee::make_simple(vec![name1, name2]);
+    let committee = Committee::make_simple(vec![validator1_pk, validator2_pk]);
```

Comment:
No need to introduce those two, since below you use `v1.public_key` anyway instead of `validator1_pk`.

```suggestion
    let committee = Committee::make_simple(vec![key1.public(), key2.public()]);
```